docs/source/esp-idf_component.rst

```diff
@@ -57,10 +84,10 @@ Option 1. Using Arduino setup() and loop()
 Option 2. Using ESP-IDF appmain()
 *********************************
 
-- You need to implement ``app_main()`` and call ``initArduino();`` in it.
+In main.c or main cpp you need to implement ``app_main()`` and call ``initArduino();`` in it.
```

Comment:
missing . (period) between main and cpp.

Reply:
Thank you for catching this. I will commit fix later if there is nothing else to be fixed.